pilot/pkg/features/experimental.go

```diff
@@ -205,4 +206,27 @@ var (
 		"If enabled, ServiceEntries with wildcard hosts and dynamic dns resolution will be allowed for TLS traffic. "+
 			"This is a security risk, susceptible to SNI spoofing, and should be used with caution. "+
 			"Only consider using this feature if the client is trusted and you understand the risks.").Get()
+
+	IgnorePilotResources = env.Register(
+		"IGNORE_PILOT_RESOURCES",
+		false,
+		"If true, pilot will ignore any Istio specific resource."+
+			"This can be used in case it is desired that Pilot watches just Gateway API resources.").Get()
+
+	bypassCRDFilter = env.Register(
```

Comment:
Per comments on the doc, I think we want a deny-list of resources, that can include a prefix match like `*.istio.io`

Reply:
yeah, I am just wondering if we want a single list, or something like "deny everything on the list A, but specific things on  list B"

Reply:
fixed to be able to use prefix match